abel/transform.py

```diff
@@ -101,7 +101,11 @@ class Transform(object):
 
     symmetry_axis : None, int or tuple
         Symmetrize the image about the numpy axis
-        0 (vertical), 1 (horizontal), (0, 1) (both axes)
+        0 (vertical), 1 (horizontal), (0, 1) (both axes). Note that the
+        abel transform is always performed around the vertical axis.
+        This parameter only affects how the image is modified before
+        (and after) applying the Abel Transform. For more information,
```

Comment:
In “Abel transform”, the word “Abel” should be capitalized (proper name), and the word “transform” should be not (common noun). Having “abel transform” and “Abel Transform” in the same paragraph looks really bizarre. :–)

Reply:
Good point. The capitalization was my fault :)